draft-ietf-quic-tls.md

```diff
@@ -616,16 +617,10 @@ the state of all streams, including application state bound to those streams.
 
 ## TLS Errors
 
-[TODO(ekr@rtfm.com): This is not going to work any more. Perhaps use
-CONNECTION_CLOSE or APPLICATION_CLOSE?)]
-
-
-Errors in the TLS connection SHOULD be signaled using TLS alerts on stream 0.  A
-failure in the handshake MUST be treated as a QUIC connection error of type
-TLS_HANDSHAKE_FAILED.  Once the handshake is complete, an error in the TLS
-connection that causes a TLS alert to be sent or received MUST be treated as a
-QUIC connection error of type TLS_FATAL_ALERT_GENERATED or
-TLS_FATAL_ALERT_RECEIVED respectively.
+If TLS experiences an error, it MUST generate an appropriate alert
+as defined in {{TLS13}}; Section 6) and then provide it to QUIC,
```

Comment:
Do we worry about ossification around these alerts? Should we have a mapping from TLS->QUIC errors?

Reply:
I don't think this is going to cause ossification. This stuff is all at least obfuscated.

Reply:
I mean clients and servers ossifying about TLS alerts as the expected error format.